codegen/src/main/twirl/templates/ScalaCommon/Marshallers.scala.txt

```diff
@@ -35,7 +36,7 @@ import pekko.grpc.PekkoGrpcGenerated
 @@PekkoGrpcGenerated
 object @{service.name}Marshallers {
   @for(serializer <- service.serializers) {
-  implicit val @serializer.name = @{service.packageName}.@{service.name}.Serializers.@{serializer.name}
+  implicit val @serializer.name: ScalapbProtobufSerializer[@{serializer.scalaType}] = @{service.packageName}.@{service.name}.Serializers.@{serializer.name}
```

Comment:
The `serializer.scalaType` refers to the newly added field here https://github.com/apache/incubator-pekko-grpc/pull/66/files#r1208112432